docs/src/literate-tutorials/ns_vs_diffeq.jl

```diff
@@ -1,3 +1,10 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+if @isdefined is_ci    #hide
+    IS_CI = is_ci      #hide
+else                   #hide
+    IS_CI = false      #hide
+end                    #hide
```

Comment:
Turns out my proposal did not work out as I thought. Fredrik suggested the following fix

https://github.com/Ferrite-FEM/Ferrite.jl/blob/c33a191f618fa9dfb2d5b5efb3c6eae8000ba6d1/docs/src/literate-tutorials/reactive_surface.jl#L1-L8